### PR TITLE
Release 2.7.1

### PR DESCRIPTION
**Pull request by d4straub**
Release 2.7.1

This is a bug fix release to fix downstream analysis with QIIME2 that failed on some systems.

Changes are:
- [#655](https://github.com/nf-core/ampliseq/pull/655) - Added `NUMBA_CACHE_DIR` to fix downstream analysis with QIIME2 that failed on some systems
- [#656](https://github.com/nf-core/ampliseq/pull/656) - Moved conda-check to script-section and replaced `exit 1` with `error()`
- [#657](https://github.com/nf-core/ampliseq/pull/657) - Corrected inaccurate reporting of QIIME2 taxonomic classifications and ASV length filtering
- [#657](https://github.com/nf-core/ampliseq/pull/657) - Improved parameter descriptions and sequence

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/dada2_taxonomy_wf.nf [hunk 1]
```diff
@@ -83,7 +83,7 @@ workflow DADA2_TAXONOMY_WF {
         //set file name prefix for SH assignments
         if (!params.skip_dada_addspecies) {
             ASV_SH_name = "ASV_tax_species_SH"
-	} else {
+    } else {
```
Comment (tillenglert) [inline subworkflows/local/dada2_taxonomy_wf.nf:86]:
```suggestion
        } else {
```

seems to be accidental?

**Review by tillenglert — APPROVED** [review]
Minor suggestion/comment otherwise LGTM!

File: subworkflows/local/dada2_taxonomy_wf.nf (hunk 1 above)
Comment (d4straub) [reply subworkflows/local/dada2_taxonomy_wf.nf:86]: Indeed, thanks!

**Review by jannikseidelQBiC — APPROVED** [review]
Looks good to me as well!

**Comment by d4straub** [comment]
Thanks!

**Pull request merged**